src/lib/form-field/form-field-legacy.scss

```diff
@@ -0,0 +1,42 @@
+@import '../core/style/variables';
+@import '../core/style/vendor-prefixes';
+
+
+// Styles that only apply to the legacy variant of the form-field.
+
+// The height of the underline.
+$mat-form-field-legacy-underline-height: 1px !default;
+
+
+.mat-form-field-variant-legacy {
+  .mat-form-field-label-wrapper {
+    filter: initial;
+  }
+
+  .mat-form-field-label {
+    backface-visibility: initial;
```

Comment:
IE doesn't support the `initial` value, we should have a lint rule for it as well. You can default it to `visible` instead.